public/src/components/App.jsx

```diff
@@ -97,24 +108,64 @@ function App() {
     }
   }
 
-  useEffect(() => {
-    axios.get(`/1`)
-      .then((result) => {
-        console.log("received data")
-        setSiteData(result.data[0])
-      })
-      .catch(err => {
-        console.log(err);
-      })
-    }, []);
+  function initialButtonClick() {
+    setShowCalendar(true)
+    setCheckInSelect(true)
+  }
+
+//PRICE CALCULATIONS
+  function calculateNights() {
+    if (checkOut) {
+      let count = 0;
+      let date = new Date(checkIn.toString());
+      let checkoutDate = new Date(checkOut.toString());
+      // while loop over check in date up to check out date
+      while (date < checkoutDate) {
+        count ++;
```

Comment:
airBnB style guide dislikes the use of unary operators. They prefer count += 1!

Reply:
ty for the heads up! I will get on that!